MAUI/DataGrid/conditional-styling.md

```diff
@@ -382,4 +385,196 @@ public class ForeColorConverter : IValueConverter
 {% endhighlight %}
 {% endtabs %}
 
-![Conditional cell styling based on CellValue in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridcellstyle_basedon_cellvalue.png)
\ No newline at end of file
+![Conditional cell styling based on CellValue in .NET MAUI DataGrid](Images/conditional-styling/maui-datagrid-conditional-datagridcellstyle_basedon_cellvalue.png)
+
+## Caption summary cell style
+
+### Conditional styling of caption summary cells using converter
+
+The appearance of caption summary cell can be customized conditionally based on [SummaryValues]() by using `converter`, where converter returns the value based on `SummaryValues`. 
+
+{% tabs %}
+{% highlight xaml %}
+    <ContentPage.Resources>
+        <local:ColorConverter x:Key="converter" />
+        <Style TargetType="syncfusion:DataGridCaptionSummaryCell">
+        <Setter Property="Background"
+                    Value="{Binding Converter={StaticResource converter}}" />
+        </Style>
+    </ContentPage.Resources>
+    <syncfusion:SfDataGrid x:Name="dataGrid" 
+                        ItemsSource="{Binding OrderInfoCollection}">
+            <syncfusion:SfDataGrid.CaptionSummaryRow>
+        <syncfusion:DataGridSummaryRow Name="CaptionSummary"
+                                ShowSummaryInRow="False"
+                                Title="Total Salary: {CaptionSummary}">
+            <syncfusion:DataGridSummaryRow.SummaryColumns>
+                <syncfusion:DataGridSummaryColumn Name="CaptionSummary"
+                                            Format="{}{Sum:C0}"
+                                            MappingName="Salary"
+                                            SummaryType="DoubleAggregate" />
+            </syncfusion:DataGridSummaryRow.SummaryColumns>
+        </syncfusion:DataGridSummaryRow>
+        </syncfusion:SfDataGrid.CaptionSummaryRow>
+    </syncfusion:SfDataGrid>
+{% endhighlight %}
+{% highlight c# %}
+public class ColorConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var summaryValue = (value as Group).SummaryDetails.SummaryValues[0];
+        var aggregateValue = summaryValue.AggregateValues.ElementAt(0);
+        var calculatedValue = aggregateValue.Value;
+
+        //custom condition is checked.
+
+        if ((double)calculatedValue < 45000)
+            return Colors.LightGreen;
+        return Colors.LightBlue;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        throw new NotImplementedException();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Here, caption summary cells are customized based on `Salary` summary value.
+
+![Conditional Style of DataGrid Caption Summary Cell using Converter](maui-datagrid-conditional-captionsummary.png)
```

Comment:
Conditional styling of caption summary cell using converter.

Reply:
modified